README.md

```diff
@@ -1,82 +1,82 @@
-=Wagn: how pioneers roll
+#Wagn: how pioneers roll
 
-==Basics
+[![Build Status](https://semaphoreapp.com/api/v1/projects/0d61c1f9-5ef0-4c5c-89c6-7664e247e4be/294670/badge.png)](https://semaphoreapp.com/ethan/wagn)
```

Comment:
Oh, I don't think we want a Semaphore badge on the README.

That badge is for core developers, but README users are generally new people trying to install.  Seems like it has a downside of advertising broken builds and no real upside.